SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -101,73 +183,98 @@ class ViewController: UIViewController {
         ])
         banner.show()
     }
+    
+    @objc func handleAddFeedbackItemRightTap() {
+        let vc = UINavigationController(rootViewController: FeedbackViewController())
+        self.present(vc, animated: true, completion: nil)
+    }
         
-    @objc func handleNavigationBarRightTap() {
-        let isEvenNumTaps = countEditTaps % 2 == 0
-        feedbackTableView.setEditing(isEvenNumTaps, animated: true)
-        countEditTaps += 1
+    @objc func handleSearchItemRightTap() {
+        searchController.delegate?.willPresentSearchController?(searchController)
     }
     
 }
 
-// MARK: - TableViewDelegate
-extension ViewController: UITableViewDelegate {
+// MARK: - UICollectionView DataSource
+extension ViewController: UICollectionViewDataSource {
     
-    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return 75
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if collectionView == headerCollectionView {
+            return headersData.count
+        } else {
+            return searchController.isActive ? filteredFeedbackData.count : feedbackData.count
```

Comment:
will `searchController` be active if it has text in it but is not `firstResponder`? If thats the case then. this should be fine but I feel like we may want the condition to rely on the amount. of input in the searchbar

Reply:
Yes, that is the case. In terms of "amount of input", does that mean only show the filteredFeedbackData if their search query is longer than ~3 characters?

Reply:
I was thinking more of just empty vs nonempty but this answers my question, checks off to me